hera_qm/xrfi.py

```diff
@@ -1639,6 +1658,310 @@ def xrfi_run(ocalfits_file, acalfits_file, model_file, data_file, history,
         uvf.write(outpath, clobber=clobber)
 
 
+def xrfi_h3c_idr2_1_run(ocalfits_files, acalfits_files, model_files, data_files,
+                        history, xrfi_path='', kt_size=8, kf_size=8, sig_init=5.0,
+                        sig_adj=2.0, ex_ants=None, metrics_file=None,
+                        clobber=False, run_check=True, check_extra=True,
+                        run_check_acceptability=True):
+    """Run the xrfi excision pipeline used for H3C IDR2.1.
+
+    This pipeline uses the detrending and watershed algorithms above.
+    Several files are concatenated together to perform the detrending,
+    and the flags from the inner files are stored.
+    The algorithm is run on several data products: omnical gains, omnical chisq,
+    abscal gains, abscal chisq, omnical visibility solutions, renormalized chisq,
+    and the raw data. All of these, except the data, are run twice - first to
+    get an initial estimate of heavily contaminated data, and a second time
+    to get better estimate. The metrics and flags from each data product and both
+    rounds are stored in the xrfi_path (which defaults to a subdirectory, see
+    xrfi_path below). Also stored are the a priori flags and combined metrics/flags.
+
```

Comment:
Explain which input files actually get saved and how the beginning and end of the list are handled.